src/app/how-i-am-doing/how-i-am-doing.component.css

```diff
@@ -0,0 +1,191 @@
+.container {
+  width: 100%;
+  padding: 20px;
+  background-color: white;
+  margin-top: 60px;
+  margin-bottom: 60px;
+  border: 1px solid #dedede;
+  border-color: rgba(NaN, 222, 222, 0.5);
+}
+.details-box-resume {
+  display: flex;
+  flex-direction: column;
+  padding-top: 10px;
+}
+
+h3 {
+  text-transform: uppercase;
+  font-size: 22px;
+  color: #526afe;
+  margin-bottom: auto;
+  font-weight: 500;
+}
+
+p {
+  margin-bottom: auto;
+  color: #6c6c6c;
+}
+
+.hello {
+  font-size: 18px;
+  font-weight: 350;
+}
+
+.material-icons {
+  color: #526afe;
+}
+.top-box-name {
+  padding-bottom: 25px;
+}
+.box-resume {
+  display: flex;
+}
+.image-resume {
+  display: flex;
+}
+
+img {
+  width: 360px;
+  height: 400px;
+}
+
+.name {
+  width: 350px;
+}
+
+.box-address {
+  padding-bottom: 25px;
+  display: flex;
+}
+
+.address {
+  padding-left: 15px;
+}
+.address-line {
+  padding-bottom: 10px;
+  color: #383838;
+  font-size: 14px;
+  text-transform: uppercase;
+  font-weight: 400;
+}
+
+.text {
+  color: #6c6c6c;
+  font-size: 12px;
+  line-height: 1.6;
+}
+
+.text a {
+  text-decoration: underline;
+  color: #526afe;
+}
+.skills {
+  margin-bottom: -30px;
+}
+
+.profesional-skills {
+  margin-bottom: -2px;
+  margin-top: 40px;
+  text-transform: uppercase;
+  color: #565656;
+  font-weight: 500;
```

Comment:
```css
font-weight: 500;
font-size: 20px;
```
I don't see these 2 properties have any effect on the way the text is displayed. Could you double check and remove them if they are not needed?

Reply:
I checked, no need, I deleted them.